knotx-core/knotx-knot-authorization/pom.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Knot.x - Reactive microservice assembler - Action Knot Verticle
```

Comment:
Should be `Auhtorization Knot Verticle`. Please check other license headers in this module.